website/pages/docs/configuration/server.mdx

```diff
@@ -54,6 +54,9 @@ server {
   this server will handle. This can be used to restrict the evaluations that
   worker threads will dequeue for processing.
 
+- `enable_event_broker` `(bool: true)` - Specifies if this server's state store
```

Comment:
```suggestion
- `enable_event_broker` `(bool: true)` - Specifies if this server
```

Mentioning the state store seems like an implementation detail - could we just say the server is doing this?